package.json

```diff
@@ -58,8 +61,8 @@
     "mocha": "^5.0.4",
     "mocha-chrome": "^1.0.3",
     "moment": "~2.21.0",
-    "ms-rest-azure-js": "Azure/ms-rest-azure-js#GitHubReference",
-    "ms-rest-js": "Azure/ms-rest-js",
+    "ms-rest-js": "file:../ms-rest-js",
```

Comment:
Is this intentional?

Reply:
No, I just caught this. I think I just took care of it.

Reply:
Fixed in https://github.com/Azure/autorest.typescript/pull/89/commits/62b458a8b207ee26d7f3b53ea91a6d3cd0566126.